api/src/main/java/eu/coatrack/api/ProxyHealthStatus.java

```diff
@@ -22,19 +22,9 @@
 
 public enum ProxyHealthStatus {
 
-    CRITICAL("critical-state"),
-    WARNING("warning-state"),
-    OK("ok-state"),
-    IGNORE("ignore-state"),
-    NEVER_CONNECTED("");
-
-    private final String cssClassForGui;
-
-    ProxyHealthStatus(String cssClassForGui) {
-        this.cssClassForGui = cssClassForGui;
-    }
-
-    public String getCssClassForGui() {
-        return cssClassForGui;
-    }
+    CRITICAL(),
```

Comment:
If there are no parameters, you can drop the `()`

Reply:
Done